File: Website/Classes/report.php
<?php
	class Report{

		public $ID;
		public $SubmittedBy;
		public $Target;
		public $Description;
		public $Claim;
		public $Claims = array();

		function __construct($ID,$SubmittedBy,$Target,$Description,$Claim,$Claims){
			$this->ID = $ID;
			$this->SubmittedBy = $SubmittedBy;
			$this->Target = $Target;
			$this->Description = $Description;
			$this->Claim = $Claim;
			$this->Claims = $Claims;
		}
	}
?>

File: Android/Reserve/app/src/main/java/com/example/reserve/Report.java
package com.example.reserve;

import java.util.ArrayList;

public class Report {

    public int ID;
    public int SubmittedBy;
    public int Target;
    public String Description;
    public int Review_ID;
    ArrayList<String> Claims = new ArrayList<String>();

    public Report(int ID, int SubmittedBy, int Target, int Review_ID, String Description, ArrayList<String> Claims){
        this.ID = ID;
        this.SubmittedBy = SubmittedBy;
        this.Target = Target;
        this.Review_ID = Review_ID;
        this.Description = Description;
        this.Claims = Claims;

    }
}
